pkg/js/transformer/dsltojson.ts

```diff
@@ -321,6 +342,13 @@ class OpenFgaDslListener extends OpenFGAListener {
       expression: "",
       parameters: {},
     };
+
+    if (this.isModularModel) {
+      //@ts-expect-error
```

Comment:
Can we get a comment here for documentation?

Reply:
This is removed up now in the [commit](https://github.com/openfga/language/pull/178/commits/4707d4906f236a0d122b7a15af1e553a8445d4d3#diff-a3aecc6cfca045d5430fdd3f9c2aadd654954d204db5068a4cbb91f1e1716fe9L347) that moved to a js-sdk generated from the new protobuf definitions so the metadata fields now exist